src/Event/SpalpAppIdsAlterEvent.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Drupal\spalp\Event;
+
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * Event that is fired when there is request for collecting available app ids.
+ */
+class SpalpAppIdsAlterEvent extends Event {
+
+  const EVENT_APP_IDS_ALTER = 'spalp_app_ids_alter';
+
+  /**
+   * List of app ids.
+   */
+  public $ids;
+
+  /**
+   * Constructs the object.
+   */
+  public function __construct($ids) {
+    $this->ids = $ids;
+  }
+
+}
```

Comment:
try to get in the habit of using getters and setters..

see examples below

Event Getter / Setter:
https://github.com/nodespark/elasticsearch_connector/blob/8.x-5.x/src/Event/BuildSearchParamsEvent.php#L35

https://github.com/nodespark/elasticsearch_connector/blob/8.x-5.x/src/Event/BuildSearchParamsEvent.php#L44